apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -5,15 +5,15 @@ import styled from '@emotion/styled'
 import { Checkbox, Col, Form, Input, Row, Typography } from 'antd'
 import React from 'react'
 import { ITicketFormState } from '@condo/domains/ticket/utils/clientSchema/Ticket'
-import { colors } from '../../constants/style'
-import { FormWithAction } from '../../containers/FormList'
-import { searchEmployee, searchProperty, searchTicketClassifier } from '../../utils/clientSchema/Ticket/search'
-import { FocusContainer } from '../FocusContainer'
-import { GraphQlSearchInput } from '../GraphQlSearchInput'
-import { LabelWithInfo } from '../LabelWithInfo'
-import { PhoneInput } from '../PhoneInput'
-import { UnitNameInput } from '../UnitNameInput'
-import { UserNameField } from '../UserNameField'
+import { colors } from '@condo/domains/common/constants/style'
+import { FormWithAction } from '@condo/domains/common/components/containers/FormList'
+import { searchEmployee, searchProperty, searchTicketClassifier } from '../../utils/clientSchema/search'
+import { FocusContainer } from '@condo/domains/common/components/FocusContainer'
+import { GraphQlSearchInput } from '@condo/domains/common/components/GraphQlSearchInput'
+import { LabelWithInfo } from '@condo/domains/common/components/LabelWithInfo'
+import { PhoneInput } from '@condo/domains/common/components/PhoneInput'
+import { UnitNameInput } from '@condo/domains/common/components/UnitNameInput'
+import { UserNameField } from '@condo/domains/common/components/UserNameField'
```

Comment:
This is closer to the `user` rather than to non-domain (`common`) components. Similar to `UserNameField`

Reply:
Discuss later, not sure we need to refactor this now.